Runtime/Services/BacktraceApi.cs

```diff
@@ -102,7 +102,7 @@ public string ServerUrl
         /// <param name="attachments">List of attachments</param>
         /// <param name="callback">Callback</param>
         /// <returns>Server response</returns>
-        public IEnumerator SendMinidump(string minidumpPath, IEnumerable<string> attachments, IDictionary<string, string> queryAttributes, Action<BacktraceResult> callback = null)
```

Comment:
can moving from query to form attribute break anything? I guess not since it's POST and for minidump specifically?

Reply:
No it wont - we're using the same pattern in different clients - example: https://github.com/backtrace-labs/backtrace-cocoa/blob/7dacd794a9318816121eafdb7d0a6e52a8cf8e55/Sources/Features/Network/MultipartRequest.swift#L55

Reply:
ok!